scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -42,10 +49,11 @@ def cpuflags():
 
 
 def hypervisor_has_virtio(libvirt_type):
-    if (libvirt_type == "xen" or libvirt_type == "hyperv"):
-        return False
-    else:
+
+    if libvirt_type == "kvm":
```

Comment:
Would create more regressions. There is this clause:
https://github.com/SUSE-Cloud/automation/blob/master/scripts/lib/libvirt/libvirt_setup.py#L101